addons/website_slides/static/src/js/slides_course_quiz_question_form.js

```diff
@@ -14,6 +14,7 @@ var QuestionFormWidget = publicWidget.Widget.extend({
     template: 'slide.quiz.question.input',
     events: {
         'click .o_wslides_js_quiz_validate_question': '_validateQuestion',
+        'click .o_wslides_js_quiz_validate_and_add_question': '_validateAndCreateNewQuestion',
```

Comment:
This is perhaps more suggestive of the differences with validate_question, similar to the `saveAndNew` utils?

```suggestion
        'click .o_wslides_js_quiz_validate_question_and_new': '_validateQuestionAndNew',
```